src/CoreShop/Component/Order/Cart/CartModifier.php

```diff
@@ -69,10 +69,18 @@ private function resolveItem(StorageListInterface $storageList, StorageListItemI
     {
         foreach ($storageList->getItems() as $item) {
             if ($this->cartItemResolver->equals($item, $storageListItem)) {
+                $this->eventDispatcher->dispatch(
```

Comment:
better to add these here `src/CoreShop/Component/Core/Order/Modifier/CartItemQuantityModifier.php`?